tools/src/main/scala/com/nvidia/spark/rapids/tool/planparser/FilterExecParser.scala

```diff
@@ -33,13 +33,15 @@ case class FilterExecParser(
     val exprString = node.desc.replaceFirst("Filter ", "")
     val expressions = SQLPlanParser.parseFilterExpressions(exprString)
     val isAllExprsSupported = expressions.forall(expr => checker.isExprSupported(expr))
+    val notSupportedExprs = expressions.filter(expr => !checker.isExprSupported(expr))
```

Comment:
this would be more efficient just to get the list of notSupported and then if that is > 0 in size then not all of them are supported, so we can remove the check above "val isAllExprsSupported = expressions.forall(expr => checker.isExprSupported(expr))"

Reply:
done